### DISCUSSION TITLE
cm311-1s CH可以从U盘启动s905l3的armbian系统但无法写入emmc

### DISCUSSION THREAD

**Discussion by creamcroissant** [ophub/amlogic-s9xxx-armbian#1648] [Amlogic]
root@armbian:~# armbian-install -m yes -l yes
[ STEPS ] Installing Armbian to internal eMMC...
[ STEPS ] Checking dependencies...
[ INFO ] Dependency check completed. Proceeding installation...
[ STEPS ] Initializing the environment...
[ INFO ] Use mainline u-boot: [ yes ]
[ INFO ] Using ampart tool: [ yes ]
[ INFO ] Show all lists: [ yes ]
[ INFO ] Internal eMMC : [ /dev/mmcblk2 ]
[ STEPS ] Start selecting device...
--------------------------------------------------------------------------------------
ID    SOC        MODEL                               DTB                                               
--------------------------------------------------------------------------------------
101   s905d      Phicomm-N1                          meson-gxl-s905d-phicomm-n1.dtb                    
102   s905d      Phicomm-N1(DMA-thresh)              meson-gxl-s905d-phicomm-n1-thresh.dtb             
103   s905d      MECOOL-KI-Pro                       meson-gxl-s905d-mecool-ki-pro.dtb                 
104   s905d      SML-5442TW                          meson-gxl-s905d-sml5442tw.dtb                     
105   s905x      HG680P                              meson-gxl-s905x-p212.dtb                          
106   s905x      B860H                               meson-gxl-s905x-b860h.dtb                         
107   s905x      TX9                                 meson-gxl-s905x-tx9.dtb                           
108   s905x      T95,XiaoMI-3S,X96                   meson-gxl-s905x-p212.dtb                          
109   s905x      TBee                                meson-gxl-s905x-tbee.dtb                          
110   s905w      TX3-Mini                            meson-gxl-s905w-tx3-mini.dtb                      
--------------------------------------------------------------------------------------
111   s905w      W95                                 meson-gxl-s905w-p281.dtb                          
112   s905w      X96-Mini                            meson-gxl-s905w-x96-mini.dtb                      
113   s905w      X96W,FunTV,MXQ-Pro-4K               meson-gxl-s905w-x96w.dtb                          
114   s905l2     MGV2000,MGV3000,M301A               meson-gxl-s905l2-x7-5g.dtb                        
115   s905l2     E900v21E                            meson-gxl-s905l2-x7-5g.dtb                        
116   s905l2     Wojia-TV-IPBS9505                   meson-gxl-s905l2-ipbs9505.dtb                     
117   s905l3     CM311-1,HG680-LC,M401A,UNT402A      meson-gxl-s905l2-x7-5g.dtb                        
118   s905l3     UNT400G1                            meson-gxl-s905l2-x7-5g.dtb                        
119   s905l3     CM211-1                             meson-gxl-s905l3b-m302a.dtb                       
120   s905l3b    E900V22D,TY1608                     meson-gxl-s905l2-x7-5g.dtb                        
--------------------------------------------------------------------------------------
121   s905l3b    E900V21E,E900V22E                   meson-gxl-s905l3b-e900v22e.dtb                    
122   s905l3b    M302A,M304A,CM211-1,CM311-1         meson-gxl-s905l3b-m302a.dtb                       
123   s905l3b    IP103H,TY1608,ZXV10-BV310           meson-gxl-s905l3b-m302a.dtb                       
124   s905lb     BesTV-R3300L,SumaVision-Q7,MG101    meson-gxl-s905x-p212.dtb                          
125   s905lb     Q96-mini                            meson-gxl-s905x-p212.dtb                          
126   s905l      UNT402A,M201-S                      meson-gxl-s905l3b-m302a.dtb                       
201   s912       Octopus-Planet                      meson-gxm-octopus-planet.dtb                      
202   s912       Octopus-Planet(FAKE)                meson-gxm-octopus-planet.dtb                      
203   s912       H96-Pro-Plus,TX92,Z6-Plus           meson-gxm-octopus-planet.dtb                      
204   s912       Vontar-X92                          meson-gxm-x92.dtb                                 
205   s912       Phicomm-T1                          meson-gxm-phicomm-t1.dtb                          
206   s912       MECOOL-M8S-Pro-L                    meson-gxm-q201.dtb                                
207   s912       T95Z-Plus                           meson-gxm-t95z-plus.dtb                           
208   s912       TX9-Pro(3G_32G_1Gb)                 meson-gxm-tx9-pro.dtb                             
209   s912       TX9-Pro(2G_16G_100Mb)               meson-gxm-q201.dtb                                
210   s912       Nexbox-A1,A95X                      meson-gxm-nexbox-a1.dtb                           
--------------------------------------------------------------------------------------
211   s912       Nexbox-A95X-A2                      meson-gxm-nexbox-a2.dtb                           
212   s912       Tanix-TX8-MAX                       meson-gxm-tx8-max.dtb                             
301   s905x2     X96-Max-4GB,Tx5-Max                 meson-g12a-x96-max.dtb                            
302   s905x2     X96-Max-2GB,A95X-F2                 meson-g12a-x96-max-rmii.dtb                       
303   s905x2     MECOOL-KM3-4G                       meson-g12a-sei510.dtb                             
304   s905l3a    E900V22C-D,CM311-1a-CH,IP112H       meson-g12a-s905l3a-e900v22c.dtb                   
305   s905l3a    CM311-1a-YST                        meson-g12a-s905l3a-cm311.dtb                      
306   s905l3a    M401A,UNT403A,B863AV3.2-M           meson-g12a-s905l3a-m401a.dtb                      
401   a311d      Khadas-VIM3                         meson-g12b-a311d-khadas-vim3.dtb                  
402   s922x      GT-King                             meson-g12b-gtking.dtb                             
403   s922x      GT-King-Pro,X88-King                meson-g12b-gtking-pro.dtb                         
404   s922x      GT-King-Pro-H                       meson-g12b-gtking-pro-h.dtb                       
405   s922x      GT-King-Pro-Rev_A                   meson-g12b-gtking-pro-rev_a.dtb                   
406   s922x      UGOOS-AM6-Plus                      meson-g12b-ugoos-am6.dtb                          
407   s922x      Hardkernel-ODroid-N2                meson-g12b-odroid-n2.dtb                          
408   s922x      Ali-CT2000                          meson-g12b-ali-ct2000.dtb                         
501   s905x3     X96-Max+_100Mb                      meson-sm1-x96-max-plus-100m.dtb                   
502   s905x3     X96-Max+_1GB                        meson-sm1-x96-max-plus.dtb                        
503   s905x3     X96-Max+(OverClock)                 meson-sm1-x96-max-plus-oc.dtb                     
504   s905x3     X96-Max+(IP1001M)                   meson-sm1-x96-max-plus-ip1001m.dtb                
505   s905x3     X96-Max+_A100                       meson-sm1-sei610.dtb                              
506   s905x3     X96-Max+_2101                       meson-sm1-x96-max-plus-2101.dtb                   
507   s905x3     X96-Max+Q1                          meson-sm1-x96-max-plus-q1.dtb                     
508   s905x3     X96-Max+Q2,X96-Air-Q1000            meson-sm1-x96-max-plus-q2.dtb                     
509   s905x3     X96-Air-1Gb                         meson-sm1-x96-air-gbit.dtb                        
510   s905x3     X96-Air,X96-Max+100W,100Mb          meson-sm1-x96-air.dtb                             
--------------------------------------------------------------------------------------
511   s905x3     Tencent-Aurora-3Pro                 meson-sm1-skyworth-lb2004-a4091.dtb               
512   s905x3     HK1-Box,Vontar-X3                   meson-sm1-hk1box-vontar-x3.dtb                    
513   s905x3     HK1-Box(OverClock)                  meson-sm1-hk1box-vontar-x3-oc.dtb                 
514   s905x3     H96-Max-X3,Infinity-B32             meson-sm1-h96-max-x3.dtb                          
515   s905x3     H96-Max-X3(OverClock),B32           meson-sm1-h96-max-x3-oc.dtb                       
516   s905x3     Ugoos-X3                            meson-sm1-ugoos-x3.dtb                            
517   s905x3     Ugoos-X3(OverClock)                 meson-sm1-ugoos-x3-oc.dtb                         
518   s905x3     TX3-1Gb                             meson-sm1-tx3-qz.dtb                              
519   s905x3     TX3-1Gb(OverClock)                  meson-sm1-tx3-qz-oc.dtb                           
520   s905x3     TX3-100Mb                           meson-sm1-tx3-bz.dtb                              
--------------------------------------------------------------------------------------
521   s905x3     TX3-100Mb(OverClock)                meson-sm1-tx3-bz-oc.dtb                           
522   s905x3     A95XF3-Air-1Gb                      meson-sm1-a95xf3-air-gbit.dtb                     
523   s905x3     A95XF3-Air-100Mb                    meson-sm1-a95xf3-air.dtb                          
525   s905x3     Whale                               meson-sm1-x96-max-plus.dtb                        
0     Other      Customize                           Enter-custom-dtb-name                             
--------------------------------------------------------------------------------------
[ OPTIONS ] Please Input ID: 117
[ INFO ] Input Box ID: [ 117 ]
[ INFO ] Model Name: [ CM311-1,HG680-LC,M401A,UNT402A ]
[ INFO ] FDTFILE: [ meson-gxl-s905l2-x7-5g.dtb ]
[ INFO ] MAINLINE_UBOOT: [  ]
[ INFO ] BOOTLOADER_IMG:  [  ]
[ INFO ] UBOOT_OVERLOAD: [ u-boot-p212.bin ]
[ INFO ] NEED_OVERLOAD: [ no ]
[ STEPS ] Which type of filesystem do you want for your root? 
-----------------------------------------------
  ID   TYPE
-----------------------------------------------
  1    ext4
  2    btrfs
-----------------------------------------------
[ OPTIONS ] Please Input ID (1/2): 1
[ INFO ] Input Type ID: [ 1 ]
[ INFO ] Type of filesystem: [ ext4 ]
[ STEPS ] Partitioning eMMC...
[ INFO ] Start backing up the default bootloader.
4+0 records in
4+0 records out
4194304 bytes (4.2 MB, 4.0 MiB) copied, 6.08488 s, 689 kB/s
[ INFO ] Delete the existing [ 2 ] partitions.
[ STEPS ] Successfully partitioned internal eMMC with amaprt.
[ INFO ] Start create MBR and partittion.
[ INFO ] 03. Write the mybox bootloader: [ /usr/lib/u-boot/mybox-bootloader.img ]
444+0 records in
444+0 records out
444 bytes copied, 0.00516079 s, 86.0 kB/s
8191+0 records in
8191+0 records out
4193792 bytes (4.2 MB, 4.0 MiB) copied, 0.366484 s, 11.4 MB/s
[ STEPS ] Processing BOOTFS partition...
[ INFO ] Formatting BOOTFS partition...
mkfs.fat 4.2 (2021-01-31)
[ INFO ] Copying BOOTFS ...
[ INFO ] Update the [ uEnv.txt ] file.
[ INFO ] Copy [ u-boot-p212.bin ] to u-boot.emmc
[ STEPS ] Start processing the rootfs partition...
[ INFO ] Formatting ROOTFS ...
/dev/mmcblk2p2 contains a ext4 file system labelled 'ROOTFS_EMMC'
	last mounted on /ddbr/install on Sat Aug 26 17:26:36 2023
[ INFO ] Copying ROOTFS ...
[ INFO ] Copying [ etc ] ...
[ INFO ] Copying [ home ] ...
[ INFO ] Copying [ lib64 ] ...
[ INFO ] Copying [ opt ] ...
[ INFO ] Copying [ root ] ...
[ INFO ] Copying [ selinux ] ...
[ INFO ] Copying [ srv ] ...
[ INFO ] Copying [ usr ] ...

Socket error Event: 32 Error: 10053.
Connection closing...Socket close.

Connection closed by foreign host.

Disconnected from remote host(新建会话 (3)) at 21:31:09.

Type `help' to learn how to use Xshell prompt.

到[ INFO ] Copying [ usr ] ... 后ssh连接断开，但设备没有关机，重启后插着U盘能够重新连上ssh

**Comment by rodren-lion** [comment]
我也有个L3的能写入 提示成功了叫我poweroff 但重启不会进入arm也没系统启动 能ping通ip
ssh 超时
